content/logs/s3.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Archives on AWS S3
+kind: Documentation
+description: "Forward all your Datadog logs to S3 for long term storage."
+---
+
+<div class="alert alert-warning">
+This feature is currently in Limited Beta. Ask your Sales representative or Customer Success Manager to have it enabled.
+</div>
+
+## S3 Archives
+
+Configure your Datadog account to forward all the logs ingested to your own S3 bucket. This allows you to keep all the logs that you used Datadog to aggregate in long-term storage. This guide shows you how to set this up.
+ 
+## Create and Configure an S3 Bucket
+
+1. Go into your [AWS Console][1] and [create an S3 bucket][2] to send your archives to. Be careful not to make your bucket publicly readable. 
+
+2. Modify your bucket to grant write-only access to the Datadog AWS user. Do this by editing your bucket’s **permissions**, and setting the **bucket policy** with the following content (replace `<MY_BUCKET_NAME>` with the name of your bucket):
+
+    ```
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "AllowDatadogArchivesUploader",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "464622532012"
+                },
+                "Action": [
+                    "s3:PutObject",
+                    "s3:PutObjectAcl",
+                    "s3:ListMultipartUploadParts",
+                    "s3:AbortMultipartUpload"
+                ],
+                "Resource": "arn:aws:s3:::<MY_BUCKET_NAME>/*"
+            }
+        ]
+    }
+    ```
+
+    {{< img src="logs/archives/log_archives_s3_iam_policy.png" alt="IAM Policy for S3 Archives" responsive="true" popup="true" style="width:75%;">}}
+
+3. Go to your [Pipelines page in your Datadog application][3], and select the **Add archive** option at the bottom.
+
+4. Input your bucket name. Optionally input a prefix directory for all the content of your log archives. Save your archive, and you are finished. 
+
+    {{< img src="logs/archives/log_archives_s3_datadog_settings.png" alt="Set your S3 bucket info in Datadog" responsive="true" popup="true" style="width:75%;">}}
+
+From the moment that your archive settings are successfully configured in your Datadog account, all the logs that Datadog ingests are enriched by your processing pipelines and subsequently forwarded to your S3 bucket for archiving.
+
+However, after creating or updating your archive configurations, it can take several minutes before the next archive upload is attempted, so you should check your S3 bucket some time later to make sure the archives have successfully been getting uploaded from your Datadog account. 
```

Comment:
We need to make the encouragement to review the setup more emphatic. Can we reword a bit and bold it, like so?

> However, after creating or updating your archive configurations, it can take several minutes before the next archive upload is attempted, so *you should check back on your S3 bucket in 15 minutes* to make sure the archives have successfully been getting uploaded from your Datadog account. After that, if you see unexpected behavior, please reach out to [support@datadog.com](mailto:support@datadog.com) for assistance.